### PR TITLE
void

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,19 +27,22 @@
...
 - :gear: [NodeJS](https://nodejs.org/) (LTS)
 - :toolbox: [Yarn](https://yarnpkg.com/)/[Lerna](https://lerna.js.org/)
 
-###### Developer Quickstart:
+### Developer Quickstart:
 
 ```bash
 lerna bootstrap
 yarn build
+lerna run test:unit
```
Comment (nflaig) [inline README.md:40]: IMO the whole point of the quickstart is to be "quick", meaning get project setup locally and learn about the existing commands (`./lodestar --help`) to get started

I am not sure if running the tests will provide valuable information here if we assume someone is completely new to Lodestar, they might just be overwhelmed...

File: README.md [hunk 2]
```diff
@@ -27,19 +27,22 @@
...
 ./lodestar --help
 ```
 
+More tests and dev-info within [CONTRIBUTING.md](https://github.com/ChainSafe/lodestar/blob/unstable/CONTRIBUTING.md#tests).
```
Comment (nflaig) [inline README.md]: for any reference within the docs should use relative paths

```suggestion
More tests and dev-info within [CONTRIBUTING.md](/CONTRIBUTING.md#tests).
```

File: README.md (hunk 1 above)
Comment (abebeos) [reply README.md:40]: > valuable information
Tests are not valuable info for a developer code-level quick-start?

Maybe add some emojis then?

Comment (nflaig) [reply README.md:40]: > Tests are not valuable info for a developer code-level quick-start?

Like I said, we are assuming here someone is looking at Lodestar for the first time and just wants to get started. I still don't really see why they should run the tests at this point, maybe you can elaborate a bit on why you think this is important here. If someone wants to dig deeper and contribute we have a reference to `CONTRIBUTING.md`.

Let's wait and get more opinions on this

Comment (abebeos) [reply README.md:40]: > Like I said, we are assuming here someone is looking at Lodestar for the first time and just wants to get started.

There it is again: "we"

> Let's wait and get more opinions on this

Opinions do not matter where a (quasi) standard exist.

Leaving you to it, me unsubscribing from this PR, as I do not have the nerves to discuss basics.

Comment (nflaig) [reply README.md:40]: > There it is again: "we"

the "we" implies it should be a general assumption, I will be more precise next time

**Pull request closed without merging**